testing/features/components/step_definitions/disclosure_steps.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Given("the Example Disclosure component is on the page") do
+  @component = Disclosure::Example.new.tap(&:load)
+end
+
+When("I open/close the disclosure component") do
+  @component.toggle.click
+end
+
+Then("The disclosure component displays expanded content") do
```

Comment:
For those two I've tried to match what's in [public-website](https://github.com/citizensadvice/public-website/blob/develop/features/advice/public/step_definitions/search_steps.rb#L96). The problem I was having was that when I was using `.not_to be_visible` was throwing an error:
<img width="1111" alt="image" src="https://user-images.githubusercontent.com/94383569/154265481-473d4073-4b45-4b87-89a6-5410a24ed3e6.png">
What I have now works, however I want to understand why `.not_to be_visible` was not working in this case